src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -22,12 +22,12 @@ import Log from '@libs/Log';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
 import {isEmptyObject} from '@src/types/utils/EmptyObject';
-import BaseListItem from './BaseListItem';
 import type {BaseSelectionListProps, ButtonOrCheckBoxRoles, FlattenedSectionsReturn, ListItem, Section, SectionListDataType} from './types';
 
 function BaseSelectionList<TItem extends ListItem>(
     {
         sections,
+        ListItem,
```

Comment:
Maybe a default value would be nice here?

Reply:
I think it's better if the client component needs to define which ListItem will be used explicitly

Reply:
this is very good notice - even better than JSDocs, I think it would be best to allow only expected types... I'll make that change, thanks

Reply:
done